modules/git/command.go

```diff
@@ -24,10 +25,14 @@ var (
 	DefaultCommandExecutionTimeout = 60 * time.Second
 )
 
+// DefaultLocale is the default LC_ALL to run git commands in.
+const DefaultLocale = "C"
+
 // Command represents a command with its subcommands or arguments.
 type Command struct {
-	name string
-	args []string
+	name   string
+	args   []string
+	Locale string
```

Comment:
Should probably be same casing as other struct elements?

Reply:
No. It's so future users have an option to override the locale and make a particular git command run in a different Locale.

Reply:
if not currently needed imho it should be currently lowercase and when needed changed to upper

Reply:
Then we may as well get rid of it entirely. Either it's configurable or it's not.

Reply:
I mean currently it is not configurable, right?

Reply:
IMHO, I don't think it should ever be configurable. It's for Gitea to parse, so it must be `"C"`.